database/index.js

```diff
@@ -0,0 +1,83 @@
+const faker = require('faker');
+var Sequelize = require('sequelize');
+var sequelize = new Sequelize('robinhood', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql'
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+const About = sequelize.define('about', {
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  description: Sequelize.STRING(2000),
+  ceo: Sequelize.STRING,
+  employees: Sequelize.INTEGER,
+  headquarters: Sequelize.STRING,
+  founded: Sequelize.INTEGER,
+  marketCap: Sequelize.INTEGER,
```

Comment:
Does marketCap, P/E and div yield need to be a Float? (ie support decimals)?

Reply:
From my understanding with robin It just need to be random numbers but I can change the data type to some financial numbers and decimals and such!